### ISSUE TITLE
PDOX-multiple email notifications

### ISSUE THREAD

**Issue by dhenry-KCI** [kcigeospatial/Fred_Co_Land-Management#1925]
got "approved plans ready for download" email 9 times.  Why?

![image](https://user-images.githubusercontent.com/4129185/65971813-cf5f3180-e436-11e9-8a9d-177bee10a5bb.png)

MW

**Comment by dhenry-KCI** [comment]
@RS-Avolve can you look at this issue?

**Comment by RS-Avolve** [comment]
This behavior is as expected:

For Project # PP245467, "Download Notification" emails were sent to: 
Ashlye Bonomo
Mara Fitter
Dustin Henry
Juanita Miles
Ashley Reed
Robert (rsosnowski@avolvesoftware.com)
Robert (PDocsRS1@gmail.com)
Administrator Temp
Mike Wilkins

For Project # PP245467, "Applicant" group members are:
Ashlye Bonomo
Mara Fitter
Dustin Henry
Juanita Miles
Ashley Reed
Robert
Administrator Temp
Mike Wilkins
NOTE: I have since revoked my user account (PDocsRS1@gmail.com).  When I revoked the user account it removed me from the Applicant group.  This explains why there were 9 emails sent out, but currently there are only 8 Applicant group members.
